overlord/assertstate/validation_set_tracking.go

```diff
@@ -0,0 +1,130 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package assertstate
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/snapcore/snapd/overlord/state"
+)
+
+// ValidationSetMode reflects the mode of respective validation set, which is
+// either monitoring or enforcing.
+type ValidationSetMode int
+
+const (
+	Monitor ValidationSetMode = iota
+	Enforce
+)
+
+// ValidationSetTracking holds tracking parameters for associated validation set.
+type ValidationSetTracking struct {
+	AccountID string            `json:"account"`
```

Comment:
Perhaps for consistency:
```suggestion
	AccountID string            `json:"account-id"`
```

Reply:
Done.